### ISSUE TITLE
Invalid `Podfile` file:  [!] Invalid `react-native-hms.podspec` file: No such file or directory

### ISSUE THREAD

**Issue by jc4p** [100mslive/react-native-hms#853]
Hi,

Thanks so much for this library and cool product! I'm trying to use version 1.1.0 from npm and I'm getting the following error during my `pod install`:

```
[!] Invalid `react-native-hms.podspec` file: No such file or directory @ rb_sysopen - [..]/node_modules/@100mslive/react-native-hms/sdk-versions.json.
```

I checked and the `sdk-versions.json` file definitely doesn't exist on my node_modules checkout or on the file browser on npm, but I see it in the github repo here.

In the meantime I'm using 1.0.0 on npm and it's working great, but wanted to let you know.

Thanks!
Kasra

**Comment by ygit** [comment]
hey @jc4p 
Thanks for reporting the issue. We have found the reason & are releasing a patch version in some time.

**Comment by ygit** [comment]
hey @jc4p 
We have released a new package version [1.1.0 ](https://www.npmjs.com/package/@100mslive/react-native-hms) which fixes this issue.

Try it out & let us know.

**Comment by jc4p** [comment]
@ygit Confirming that `pod install` works now, thank you!
